public/app/plugins/datasource/grafana-azure-monitor-datasource/components/VariableEditor/VariableEditor.tsx

```diff
@@ -286,6 +288,26 @@ const VariableEditor = (props: Props) => {
           />
         </InlineField>
       )}
+      {query.queryType === AzureQueryType.AzureResourceGraph && (
+        <>
+          <ArgQueryEditor
+            subscriptionId={datasource.azureLogAnalyticsDatasource.defaultSubscriptionId}
+            query={query}
+            datasource={datasource}
+            onChange={onQueryChange}
+            variableOptionGroup={variableOptionGroup}
+            setError={setError}
+          />
+          {errorMessage && (
+            <>
+              <Space v={2} />
+              <Alert severity="error" title="An error occurred while requesting data from Azure Monitor">
+                {errorMessage}
+              </Alert>
+            </>
+          )}
```

Comment:
you don't need this block (the error is catched elsewhere)

Reply:
Actually that seems to be used for errors when getting subscriptions so I'll leave it in both